sampling/main.py

```diff
@@ -21,6 +21,9 @@
 parser.add_argument("--bias",type=int,default=0,help="bias the relative error values")
 parser.add_argument("--seed",type=str,default=0,help="seed for shuffling of uniform leaf list")
 parser.add_argument("sample_number",type=int,help="number of samples")
+parser.add_argument("--confidence_level",type=float,default=0.90,help="How strong should the confidence level of the confidence interval be? Values between 0 and 1")
+parser.add_argument("--confidence_noplot",help="Should the confidence interval be hidden in the plot?",action="store_true")
+parser.add_argument("--stds_not_weighted",help="Should the confidence interval be weighted by progress of tree exploration?",action="store_true")

```

Comment:
My suggestion: Command line arguments should not have underscores because it is annoying to type. Change them to hyphens '-'. As python variables (members of the `args` object), they must be stored with underscores again. For this, there is a keyword `dest=stds_not_weighted` for the `add_argument` function.